tests/parsers/viminfo.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""Tests for the Viminfo parser."""
+
+import unittest
+
+from plaso.parsers import viminfo
+from plaso.containers import warnings
+
+from tests.parsers import test_lib
+
+
+class ViminfoParserTest(test_lib.ParserTestCase):
+  """Tests for the Viminfo parser."""
+
+  # pylint: disable=protected-access
+
+  def testParse(self):
+    """Tests the Parse function."""
+    parser = viminfo.VimInfoParser()
+    storage_writer = self._ParseFile(['.viminfo'], parser)
+
+    number_of_events = storage_writer.GetNumberOfAttributeContainers('event')
+    self.assertEqual(number_of_events, 10)
+
+    extraction_warnings = list(storage_writer.GetAttributeContainers(
+        warnings.ExtractionWarning.CONTAINER_TYPE))
+
+    self.assertEqual(
+        extraction_warnings[0].message,
+        'unable to parse log line: \'> ~\\\\_vimrc\'')
+    self.assertEqual(
+        extraction_warnings[3].message,
+        'unable to parse log line: '
+        '\'> C:\\\\Program Files (x86)\\\\Vim\\\\.vimrc\'')
+
+    number_of_warnings = storage_writer.GetNumberOfAttributeContainers(
```

Comment:
Please add a note that the warnings are about unsupported format features.

Reply:
also please move this before the test of the individual warning

Reply:
Done